### ISSUE TITLE
UI for resources configurations for pipeline build pods

### ISSUE THREAD

**Issue by gitlawr** [rancher/rancher#17569]
**What kind of request is this (question/bug/enhancement/feature request):**
enhancement

**Description**
Under project-tools-pipeline settings, Be able to configure resouces(CPU/memory request/limit) for dynamically provisioned build pods.

Related issue: https://github.com/rancher/rancher/issues/16538

**Comment by tfiduccia** [comment]
@gitlawr - I'm not sure what I should be seeing.  Can you make a screenshot?  I went to settings for my pipeline and only saw this:
![image](https://user-images.githubusercontent.com/11514927/52074974-5b3b5980-2548-11e9-90d7-644dcaa20b70.png)

**Comment by loganhz** [comment]
UI PR is not merged yet. We added it in the `Tools` => `Pipeline`

**Comment by westlywright** [comment]
Had to roll back some changes on master this will be on hold until we can merge again.

**Comment by davidnuzik** [comment]
I do not like that when we authenticate such as with github and pick our repository (ies) we get routed back to listing of repos I think it would make more sense to route back to pipeline settings. It's not as obvious to the user that this is a new feature unless we do that and when they set up their pipeline the first time they would likely want to see any additional settings/options they can pick.
I created #17942 to track this enhancement request.

**Comment by davidnuzik** [comment]
Version: Master 2.2 (2/8/19)
Validated new feature - Pipeline Settings. Tested with GitHub.
Confirm form works / looks good. Validate the various options (http/api).
- Inspect pipeline executors, when you + or - a PUT updates this live without form submission (good).
- Inspect resource limit/reservation options. When you click the "Update Limit and Request" button a PUT is submitted for each endpoint (good).
- Inspect custom CA cert. When you click "Save cacerts" PUT is submitted and content looks good (fuzz, test newlines, etc).
- Confirm "Disable Repository Authentication" works. It will also ask user to confirm with a new button "Are you sure to disable repository authentication?" if you do not click it after about 10 seconds it will revert back to the normal "Disable Repository Authentication" button. If you do click it it will successfully disable repo auth and bring us back to our options for GitHub, GitLab, BitBucket.

During my testing I noticed that we could also use some minor fixes to English translations so I also created #17943
